_posts/2014-08-01-accounts.markdown

```diff
@@ -22,9 +22,17 @@ account creation.
 | `user:first_name`             | String            |           |          |         | First name of the primary payroll administrator
 | `user:last_name`              | String            |           |          |         | Last name of the primary payroll administrator
 | `user:email`                  | String            |           |          |         | Email address for primary payroll administrator. Will also be used as their login.
+| `user:phone`                  | String            |           |      Yes |         | Phone number for primary payroll administrator.
 | `company`                     | Object            |           |          |         | Information for the new company
 | `company:name`                | String            |           |          |         | Company name. Must be at least two characters long.
+| `company:trade_name`          | String            |           |      Yes |         | Company legal name.
+| `company:ein`                 | String            |           |      Yes |         | Company's EIN (aka FEIN)
 | `company:states`              | Array             |           |          |         | List of states (two letter abbreviations) in which the company operates.
+| `partner`                     | Object            |           |      Yes |         | Partnership Information
+| `partner:name`                | String            |           |          |         | Name of your application as registered for Gusto API access
+| `partner:company_id`          | String            |           |          |         | Company ID on your platform, for linking with equivalent Gusto entity
+| `partner:user_id`             | String            |           |          |         | User ID on your platform, for linking with equivalent Gusto entity
+| `partner:accounting_firm_id`  | String            |           |          |         | ID of Accounting Firm that manages the company on your platform.
```

Comment:
Two things:

1. We should replace `Yes` with `X` because this is what other pages in our API docs use

<img width="892" alt="screen shot 2018-10-16 at 2 10 48 pm" src="https://user-images.githubusercontent.com/21319304/47047895-c3a72c00-d14d-11e8-8b50-d6a20f8fcd1c.png">

2. Would the fields on lines 32-35 be optional as well? Or if `partner` is present, does it expect to see all 4 of those attributes? If they are optional within `partner` parameter, then I would mark them with an `X` too. If they are not optional, I would add a note next to `Partner information` stating that if `partner` param is present then other fields must exist within as well.